.swiftformat

```diff
@@ -32,5 +32,23 @@
 --disable redundantInit
 --disable void
 
+# These are rules that were all added to swiftformat when we upgraded from 0.44 to 0.53.
```

Comment:
This is a non-automatic change. What I did is to create a list of all the changes `swiftformat` wanted to apply; sorted them; and then figured out which of those rules were new to the latest version. Here's what I ended up with:

```
   1 assertionFailures - new
   3 blankLineAfterImports - new
   4 blankLinesAtStartOfScope - old
  52 elseOnSameLine - old
   5 emptyBraces - old
   9 enumNamespaces - new
 155 extensionAccessControl - new
   2 hoistTry - new
 179 indent - old
 147 modifierOrder - new
  81 preferForLoop - new
 105 redundantClosure - new
   7 redundantFileprivate - old 
 317 redundantInternal - new
  30 redundantParens - old
   6 redundantReturn - old
  51 redundantSelf - old 
 122 redundantType - new
   5 redundantVoidReturnType - old
   4 trailingSpace - old
   1 unusedArguments - old
 136 wrap - old
   2 wrapLoopBodies  - new
 242 wrapMultilineStatementBraces - new
  30 wrapSingleLineComments - new
```